src/plugins/ardura/Subhoofer/2.2.2.yaml

```diff
@@ -0,0 +1,26 @@
+---
+name: Subhoofer
+author: Ardura
+homepage: https://github.com/ardura/Subhoofer
+description: Sub and Bass enhancement plugin
+date: 2024-08-02T07:00:00.000Z
+license: gpl-3.0
+tags:
+  - Effect
+  - Distortion
+files:
+  audio:
+    name: https://studiorack.github.io/studiorack-registry/plugins/ardura/actuate/actuate.flac
```

Comment:
Should this be `actuate` or `subhoofer`?

Reply:
I don't have any audio demos for some plugins so I left them as the Actuate one heh. Are these nullable?

Reply:
If you omit the `audio` field it will show in the registry like this:
```
X airwindows/airwindows | 1.0.0 | src/plugins/airwindows/airwindows/1.0.0.yaml
- audio field missing
```

but will still be added to the registry!

Reply:
Fixed plugins that don't have audio files with them, thanks